packages.R

```diff
@@ -71,6 +71,8 @@ works_with_R(
   gridExtra="2.3",
   neuroblastoma="1.0",
   tikzDevice="0.12.3.1",
+  microbenchmark="1.4.7",
+  directlabels="2021.2.24",
```

Comment:
please remove this directlabels line, use the tdhock/directlabels@hash line above instead (more recent)